src/huggingface_hub/hf_api.py

```diff
@@ -340,7 +340,8 @@ class CommitInfo(str):
     pr_num: Optional[str] = field(init=False)
 
     # legacy url for `str` compatibility (ex: url to uploaded file, url to uploaded folder, url to PR, etc.)
-    _url: str = field(repr=False, default=None)  # type: ignore  # defaults to `commit_url`
+    # type: ignore  # defaults to `commit_url`
+    _url: str = field(repr=False, default=None)
```

Comment:
```suggestion
    _url: str = field(repr=False, default=None)  # type: ignore  # defaults to `commit_url`
```
This should be on a single-line to avoid mypy complaining